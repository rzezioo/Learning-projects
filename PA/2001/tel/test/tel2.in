100 107
9 65 8 21 58 78 53 5 76 67 82 67 80 85 86 25 66 62 85 24 101 40 57 72 27 37 56 63 100 17 61 40 81 68 60 31 78 44 35 46 3 49 5 82 26 90 38 24 83 15 47 9 94 35 12 13 71 106 7 102 16 68 34 28 67 25 98 37 107 64 82 2 5 18 15 70 1 52 93 15 105 32 23 91 66 74 35 68 72 42 62 19 41 27 47 107 51 37 37 50
23 10 17 26 11 86 70 94 88 47 1 24 81 55 65 44 2 2 74 80 43 46 94 65 21 5 14 67 19 46 31 93 7 47 18 17 84 39 10 23 85 63 46 65 17 62 60 70 64 33 49 58 30 94 23 2 98 88 68 68 33 50 13 91 48 82 8 83 21 69 57 57 31 3 74 99 16 33 68 31 18 69 89 99 62 63 1 12 50 20 31 34 22 43 24 69 25 83 52 45 52 8 1 34 62 26 33
