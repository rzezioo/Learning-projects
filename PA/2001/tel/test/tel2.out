0001110010101010000001000011100100111101111110001010110101111011011000111011111010000010110100111001
00010101111101001101011100101101100111101010100100001110101011011111101011101010100110111111001110101011000
